microsoftbotframework/conversationoperations.py

```diff
@@ -69,7 +69,7 @@ class CreateConversation(Activity):
     def __init__(self, **kwargs):
         super(CreateConversation, self).__init__(**kwargs)
 
-    def send(self):
+    def send(self, timeout_seconds):
```

Comment:
Please remove the timeout_seconds argument, it must be passed in at class instantiation per your other additions